services/cd-service/pkg/service/service.go

```diff
@@ -83,19 +83,25 @@ func (s *Service) ServeHTTPRelease(tail string, w http.ResponseWriter, r *http.R
 	}
 	if err := r.ParseMultipartForm(MAXIMUM_MULTIPART_SIZE); err != nil {
 		w.WriteHeader(400)
-		fmt.Fprintf(w, "invalid body: %s", err)
+		fmt.Fprintf(w, "Invalid body: %s", err)
 		return
 	}
 	form := r.MultipartForm
 	if len(form.Value["application"]) != 1 {
-		w.WriteHeader(400)
-		fmt.Fprintf(w, "invalid app name")
-		return
+		if len(form.Value["application"]) > 1 {
+			w.WriteHeader(400)
+			fmt.Fprintf(w, "Please provide single application name")
+			return
+		} else {
+			w.WriteHeader(400)
+			fmt.Fprintf(w, "Invalid application name")
+			return
+		}
 	}
 	application := form.Value["application"][0]
 	if !valid.ApplicationName(application) {
 		w.WriteHeader(400)
-		fmt.Fprintf(w, "invalid app name")
+		fmt.Fprintf(w, "Invalid application name")
```

Comment:
please give some more details here in this error, like
```suggestion
		fmt.Fprintf(w, "Invalid application name: '%s' - must match regexp '%s' and less than %d characters", regexp, application, maxCharacters)
```